src/modules/extendedBuilding/i18n/en_AU.json

```diff
@@ -0,0 +1,30 @@
+{
+    "enhancedPersonnelAssignment": {
+        "toggleFittingPersonnel": "Hide personnel without suitable training"
+    },
+    "expansions": {
+        "notBuild": "Not built yet"
+    },
+    "fastDispatchChooser": {
+        "noDispatch": "No control center"
+    },
+    "personnelDemands": {
+        "additional": "Values in brackets exclude vehicles in FMS 6.",
```

Comment:
Can we try to use "status" instead of "FMS" or a they using the abbreviation also ingame?